src/plugins/condition/components/ConditionEdit.vue

```diff
@@ -259,20 +324,18 @@ export default {
                 this.condition.definition.output = this.selectedOutputKey;
             }
         },
-        operationKeyChange(ev) {
-            if (ev.target.value !== 'isUndefined' && ev.target.value !== 'isDefined') {
+        setInputValueVisibility(ev) {
+            if (this.selectedOperationKey !== 'isUndefined' && this.selectedOperationKey !== 'isDefined') {
```

Comment:
This is hard to follow. Are you trying to check for the value not being undefined? You can use the typeOf operator like so `if (typeOf this.selectedOperationKey !== 'undefined')`

Reply:
No, it's comparing the 'name' property of the operation is 'isUndefined'. 
See this file: https://github.com/nasa/openmct/blob/set-criteria-options/src/plugins/condition/utils/operations.js#L159

Reply:
In that case, i would expect the variable to be selectedOperationName.

Reply:
Done.